src/lib/Behat/PageElement/ElementFactory.php

```diff
@@ -43,8 +43,12 @@ private static function getFactory(int $installType): ElementFactory
         switch ($installType) {
             case InstallType::PLATFORM:
                 return new PlatformElementFactory();
+            case InstallType::PLATFORM_DEMO:
```

Comment:
I like the form:
```
            case InstallType::PLATFORM:
            case InstallType::PLATFORM_DEMO:
                 return new PlatformElementFactory();
             case InstallType::ENTERPRISE:
             case InstallType::ENTERPRISE_DEMO:
                 return new EnterpriseElementFactory();
```
better, for me it clearly shows that we're using only two factories.